devicemodel/hw/platform/cmos_io.c

```diff
@@ -88,10 +89,15 @@ cmos_io_handler(struct vmctx *ctx, int vcpu, int in, int port, int bytes,
 			return -1;
 		}
 
-		if (in)
-			*eax = cmos_buffer[buf_offset];
+		if (in) {
+			*eax = ctx->cmos_buffer[buf_offset];
```

Comment:
i'm confused here. *eax can hold 256 bit data?

by the way, can you let me know where is the code or patch in vSBL to retrieve/read vRPMB key from CMOS? pretty thanks.